dcos/mesos.py

```diff
@@ -30,14 +30,10 @@ class DCOSClient(object):
 
     def __init__(self):
         config = util.get_config()
```

Comment:
we need to call this `toml_config` or something since we are using a method from the config module on line 35, and the config variable is overwriting the module import name in this method